example/src/SiteWrapper.react.js

```diff
@@ -17,7 +17,12 @@ type Props = {|
 |};
 
 type State = {|
-  unreadCount: number,
+  notificationsObjects: Array<{
```

Comment:
`Notification.react` exports its props, so could we potentially import this instead of redefining the type here?  Not 100% sure if this is doable but maybe something like: 

`import type { Props as NotificationProps } from ...`
Then you can type this as `notificationsObjects: Array<NotificationProps>,`

Reply:
Its doable, I would need to 

a) import the Notification file directly, or 
b) export the type from index.js 

(I already tried this but as nothing else had this pattern I thought I would type a similar object). Do you have a preference?

Reply:
Let's export the type from index.js similar to lines 4-18 in that file (exporting event types).